app/views/posts/_expanded.html.erb

```diff
@@ -293,6 +293,33 @@
           </div>
         <% end %>
 
+        <% if is_question && post.children.length >= SiteSetting["TableOfContentsThreshold"] %>
+          <div class="toc" id="toc-example-toggle">
```

Comment:
[**nitpick (non-blocking):**](https://conventionalcomments.org/) I don't know if that's just me, but the TOC feels somehow off. Please consider adding some left margin, so that it aligns better with the post tools and the comments.

![screenshot demonstrating the issue](https://user-images.githubusercontent.com/21335202/98607955-7a087b80-22ea-11eb-867a-d1136dc70043.png)